src/Plugin.php

```diff
@@ -121,8 +146,16 @@ public function handleShortenEvent($url, Deferred $deferred)
             $deferred->resolve($url);
             return;
         }
-        $this->logDebug('[' . $requestId . ']Shortening url: ' . $url);
 
+        // Check if URL host is one we're skipping
+        $host = parse_url($url, PHP_URL_HOST);
```

Comment:
May want to add a `\` in front of `PHP_URL_HOST` as we've found that not being specific about constants or classes being in the root namespace can cause issues with HHVM compatibility.